packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -148,6 +148,16 @@ export function createOdspNetworkError(
             }
             break;
         case 404:
```

Comment:
I do not see in this PR passing a header to tell SPO to return redirect info (and not use 308). Is it missing?